internal/auth0/client/resource.go

```diff
@@ -514,6 +515,98 @@ func NewResource() *schema.Resource {
 								},
 							},
 						},
+						"azure_blob": {
+							Type:        schema.TypeList,
+							Optional:    true,
+							Computed:    true,
+							MaxItems:    1,
+							Description: "Azure Blob Storage Addon configuration.",
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"account_name": {
+										Description: "Your Azure storage account name. Usually first segment in your " +
+											"Azure storage URL. e.g. `https://acme-org.blob.core.windows.net` would " +
```

Comment:
```suggestion
											"Azure storage URL, for example `https://acme-org.blob.core.windows.net` would " +
```
The docs team's guidelines instruct not to use latin abbreviations.